source/installing.rst

```diff
@@ -30,7 +30,7 @@ commands that require elevated priviges start with |sudo| in the following proce
 
 .. code-block:: bash
 
-   $ sudo percona-release setup ppg-11
+   $ sudo percona-release setup ppg11
```

Comment:
I am not sure but may be we should also add some note about enabling standalone repo somewhere in doc `sudo percona-release enable ppg-11 release`